grails-app/views/order/_orderSummary.gsp

```diff
@@ -76,10 +76,10 @@
 
             <tbody>
 
+%{--            <g:set var="orderItemsDerivedStatus" value="${orderInstance?.getOrderItemsDerivedStatus()}"/>--}%
             <g:each var="orderItem" in="${orderInstance?.orderItems?.sort { a,b -> a.dateCreated <=> b.dateCreated ?: a.orderIndex <=> b.orderIndex }}" status="i">
-                <g:set var="isItemCanceled" value="${orderItem.orderItemStatusCode == OrderItemStatusCode.CANCELED}"/>
-                <g:if test="${!isItemCanceled || orderInstance?.orderType==OrderType.findByCode(OrderTypeCode.PURCHASE_ORDER.name())}">
-                    <tr class="order-item ${(i % 2) == 0 ? 'even' : 'odd'} dataRow" style="${isItemCanceled ? 'background-color: #ffcccb;' : ''}">
+                <g:if test="${!orderItem?.canceled || orderInstance?.isPurchaseOrder}">
+                    <tr class="order-item ${(i % 2) == 0 ? 'even' : 'odd'} dataRow" style="${orderItem?.canceled ? 'background-color: #ffcccb;' : ''}">
                         <td>
                             <div class="tag ${orderItem?.canceled ? 'tag-danger' : ''}">
                                 <format:metadata obj="${orderItem?.canceled ? orderItem?.orderItemStatusCode?.name() : orderItem?.getOrderItemStatus()}"/>
```

Comment:
There's a call to getOrderItemStatus() here as well.

Reply:
oof I missed that one

Reply:
wait, i pushed "non-fixed" version :see_no_evil:

Reply:
@jmiranda Fixed